src/email-sender.js

```diff
@@ -0,0 +1,39 @@
+const nodemailer = require('nodemailer');
+
+/*
+*                             HOW TO USE                                 *
+* Import this file - const sendEmail = require('./email-sender);         *
+* send email - sendEmail.sendMessage("Put your error message in here");  *
+*/
+
+exports.sendMessage = async function (errorMessage) {
```

Comment:
We need to add a way to specify a few things, at the very least:

1. To
1. Subject
1. Body

I would put these in as params to the function.

Reply:
Another thing you need to do, is properly wrap the callback style of nodemailer into a Promise:

```js
// This needs to return a Promise
async function doSomething() {
  return new Promise((resolve, reject) => {
    ...
    sendMail(options, function(err, info) {
        if(err) reject(err);
        else resolve(info);
    });
  });
}
```

or you can do it using [util.promisify()](https://nodejs.org/dist/latest-v8.x/docs/api/util.html#util_util_promisify_original)